src/main/java/nl/hsac/fitnesse/junit/HsacFitNesseRunner.java

```diff
@@ -101,6 +102,9 @@ protected FitNesseContext createContext(Class<?> suiteClass) throws Exception {
     protected void runPages(List<WikiPage> pages, RunNotifier notifier) {
         boolean seleniumConfigOverridden = configureSeleniumIfNeeded();
         try {
+            if ("true".equalsIgnoreCase(System.getProperty(allureEnablerVariableName))){
```

Comment:
We need the page tags in the listener to support grouping by tag in the report. This is now solved by instantiating the listener and giving it the pages object. if we can achieve the same by adding it to the pom, that would be the preferred way, but I don't think we can get the tags to the listener in that case?